### ISSUE TITLE
[tutorials] Missing css import

### ISSUE THREAD

**Issue by vernonk** [lit/lit.dev#691]
On Step 7/8 of the tutorial the provided code is missing the `css` import. While this is pretty easy to figure out, it may cause some to stumble as there is no reference in the tutorial that it needs to be imported. Without it, the preview area just renders blank.

**Comment by AndrewJakubowicz** [comment]
Investigated. Thank you for the report. This bug occurs in the `js` version of step 7.

Reproduction steps:
 1. Navigate to https://lit.dev/tutorial/#06
 2. Make sure the language is set to `JavaScript`.
 3. Press `Reset` button.
 4. Note that there is no `css` import included. (Compared to TypeScript).

**Comment by AndrewJakubowicz** [comment]
cc @aomarks - I believe this happens because the ts-transformer strips any unreferenced imports.

Is the solution for this case to update `ts-transformer`, or make a `js` file version for this step?

**Comment by aomarks** [comment]
Yes that's right, TYpeScript will remove unused imports. If you want a `before` step that has imports that aren't used, you can just create the `.js` version in the same directory, and then the TS->JS generator will know to skip trying to auto-generate that one.

For example we do that in this step: https://github.com/lit/lit.dev/tree/main/packages/lit-dev-content/samples/tutorials/intro-to-lit/01/before
